src/PaymentGateways/Gateways/Stripe/Traits/CanSetupStripeApp.php

```diff
@@ -14,8 +16,11 @@ trait CanSetupStripeApp
      *
      * @return void
      */
-    protected function setupStripeApp($formId)
+    protected function setupStripeApp(int $formId)
```

Comment:
Since you're adding types, include `: void` and clean up the docblock.

Reply:
fixed: https://github.com/impress-org/givewp/commit/3d9b36cc3e7369d2794a661f70a4da8216bdc0a7

Reply:
Still not seeing `: void` on the method signature.

Reply:
@ravinderk I removed `@return void` from the list of files edited in this pull request.
ref: https://github.com/impress-org/givewp/compare/62737bb23d7f...7b7148a7802d

Reply:
@JasonTheAdams I get you wrong here. Removed `@return void` type 😵‍💫

Reply:
@JasonTheAdams are you using 7.1?